src/wp-includes/post.php

```diff
@@ -338,9 +338,8 @@ function create_initial_post_types() {
 
 	$template_edit_link = 'site-editor.php?' . build_query(
 		array(
-			'postType' => '%s',
-			'postId'   => '%s',
-			'canvas'   => 'edit',
+			'p'      => str_replace( '/', '%%2F', '/%s/%s' ),
```

Comment:
```suggestion
			'p'      => '/%s/%s',
```

I saw [your comment](https://core.trac.wordpress.org/ticket/63110#comment:7). It might be good to use slashes for now.